index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Survey form</title>
+</head>
+<body>
+    <h1 id ="title">Microvres Survey Form</h1>
```

Comment:
Incorrect spellings for Microverse. Make sure you use correct spelling in your code to make it look like professional.